.github/actions/setup-node-npm/action.yml

```
@@ -0,0 +1,25 @@
+name: Setup Node and NPM
+
+description: Setup Node and NPM with caching
```

Nit: The convention is to use lower case.

```suggestion
name: Setup Node.js and npm

description: Setup Node.js and npm with caching
```